external/mmdetection/detection_tasks/extension/datasets/mmdataset.py

```diff
@@ -121,17 +121,19 @@ def __len__(self):
         def __getitem__(self, index):
             """
             Prepare a dict 'data_info' that is expected by the mmdet pipeline to handle images and annotations
-            :return data_info: dictionary that contains the image and image metadata, as well as the labels of the objects
-                in the image
+            :return data_info: dictionary that contains the image and image metadata, as well as the labels of
+            the objects in the image
             """
 
             dataset = self.ote_dataset
             item = dataset[index]
+            label_idx = {label.id: i for i, label in enumerate(self.labels)}
```

Comment:
This line does not need to be run for each get_item() call.
Please move it to init() or somewhere and then reuse label_idx.

Reply:
Move label_idx to init and set self.label_idx